assembl/static2/js/app/components/administration/sectionTitle.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const SectionTitle = ({ i18n, phase, tabId, annotation }) => {
+  const { locale, translations } = i18n;
+  return (
+    <div>
+      <h3 className="dark-title-3">
+        {translations[locale].administration[phase][tabId]}
```

Comment:
No, we are sure the locale is in translations. See the getLocale function in globalFunctions.js